### PR TITLE
docs(faqs): Update and rename faq

### PR DESCRIPTION
**Pull request by vladislav-terzi**
Edited a name of the FAQ file, as well as the name in the doc itself

Required for merge:
- [x] `pr-workflow-check / draft-release-check` is ✅ success
- Other rules GitHub shows you, or can be read in [configuration](../terraform/github.com/branches.tf) 

Makes review faster:
- [x] PR title is my best effort to provide summary of changes and has clear text to be part of release notes 
- [ ] I marked PR by `misc` label if it should not be in release notes
- [x] Linked Zenhub/Github/Slack/etc reference if one exists
- [x] I was clear on what type of deployment required to release my changes (node, runtime, contract, indexer, on chain operation, frontend, infrastructure) if any in PR title or description
- [x] Added reviewer into `Reviewers`
- [x] I tagged(`@`) or used other form of notification of one person who I think can handle best review of this PR
- [x] I have proved that PR has no general regressions of relevant features and processes required to release into production
- [x] Any dependency updates made, was done according guides from relevant dependency
- Clicking all checkboxes 
- Adding detailed description of changes when it feels appropriate (for example when PR is big)

### CODE REVIEW

File: docs/docs/user-guides/trustless-faq.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# FAQ
+# Trustless FAQ
```
Comment (JafarAz) [inline docs/docs/user-guides/trustless-faq.md:1]: Are these FAQ's solely for app.trustless.zone?

Comment (vladislav-terzi) [reply docs/docs/user-guides/trustless-faq.md:1]: yes, this one is for trestles only. we plan on adding more FAQs later

**Review by JafarAz — DISMISSED** [review]

**Comment by vladislav-terzi** [comment]
@dzmitry-lahoda can you merge this one? Or @JafarAz

**Comment by vladislav-terzi** [comment]
@RustNinja can you pls merge? Passed all the tests, just waiting for the review

**Pull request closed without merging**